sse_client/event.go

```diff
@@ -0,0 +1,119 @@
+package sse_client
```

Comment:
I've must miss it when I search for SSE implementation. Unfortunately I think we need to stick with custom implementation. We need to detect if we are connected to master. And the only way to do it is to check response headers.